modules/manage/pages/kubernetes/security/tls/use-cert-manager.adoc

```diff
@@ -0,0 +1,412 @@
+= Use cert-manager to manage TLS certificates
+:description: Enable TLS encryption in your Redpanda cluster and use cert-manager to simplify the process of obtaining, renewing, and using certificates.
+
+Learn how to enable TLS encryption in your Redpanda cluster and use glossterm:cert-manager[^] to simplify the process of obtaining, renewing, and using certificates.
+
+When using cert-manager for TLS certificate management, you have the option to use a self-signed certificate or a certificate signed by a trusted Certificate Authority (CA). This topic provides instructions for each option.
+
+Redpanda supports both TLS and mTLS:
+
+- TLS, previously SSL, provides encryption for client-server communication. A server certificate prevents third parties from accessing data transferred between the client and server.
+
+- mTLS, or mutual TLS, is a protocol that authenticates both the server and the client. In addition to the server certificate required in TLS, mTLS also requires the client to give a certificate. mTLS is useful for environments that require additional security and only have a small number of verified clients.
+
+== Prerequisites
+
+You must have the following:
+
+* Kubernetes cluster: Ensure you have a running Kubernetes cluster, either locally, such as with minikube or kind, or remotely.
+
+* https://kubernetes.io/docs/tasks/tools/#kubectl[Kubectl^]. Ensure you have the `kubectl` command-line tool installed and configured to communicate with your cluster.
+
+* https://cert-manager.io/docs/installation/helm/[cert-manager^]. Ensure you have cert-manager and its custom resource definitions (CRDs) installed.
+
+* If you want to connect to your Redpanda cluster from outside Kubernetes, make sure to xref:manage:kubernetes/networking/external/index.adoc[enable external access].
+
+== Use a self-signed certificate
+
+By default, the Redpanda Helm chart uses cert-manager to generate four Certificate resources. These resources provide Redpanda brokers with self-signed TLS certificates for internal and external listeners.
```

Comment:
@JakeSCahill I don't understand all the things that get capitalized in Kubernetes, but this file is kinda confusing how only sometimes "Certificate" is capitalized. I lowercased "Certificate Authority" because we lowercase it in the Linux files. But please double check that these usages in your k8s files are all correct.

Reply:
If Certificate is title case it refers to the Certificate resource in Kubernetes. If it's lowercase, it refers to the general term.